context/app/static/js/components/Detail/Dataset/Dataset.jsx

```diff
@@ -61,7 +61,7 @@ function DatasetDetail(props) {
   } = assayMetadata;
 
   const shouldDisplaySection = {
-    visualization: 'name' in vitData,
+    visualization: 'name' in vitData || (vitData[0] && 'name' in vitData[0]),
```

Comment:
It scares me a bit if sometimes we'll get an object, and sometimes there's an array of objects.

Reply:
Do you have any other recommendations?  I saw it a few times when looking up how to handle it.